pom.xml

```diff
@@ -155,23 +155,30 @@
         <groupId>org.apache.tinkerpop</groupId>
         <artifactId>gremlin-core</artifactId>
         <version>${tinkerpop.version}</version>
+        <exclusions>
```

Comment:
We cannot exclude javatuples because it is used to serde `Pair` instances. We cannot exclude javapoet either because there is one annotation processor declared in gremlin-core (`GremlinDslProcessor`) that would throw an error and make the project compilation fail if javapoet is absent.

Reply:
I created [TINKERPOP-2411](https://issues.apache.org/jira/browse/TINKERPOP-2411) to suggest moving the Gremlin processor to its own artifact.